ui/pom.xml

```diff
@@ -61,6 +61,7 @@
                         </goals>
                         <configuration>
                             <arguments>install</arguments>
+                            <npmRegistryURL>${npmRegistryURL2}</npmRegistryURL>
```

Comment:
Where does this get set?  `npmRegistryURL2` is a strange property name...

Reply:
at productization level. 
the trailing 2 might have been a bad choice, but this was mainly to differentiate it from the same without the suffix, that is used by the plugin to apply at global level.

It's a bad name indeed, but the result of a copy paste that was also in the project we copied that from so I haven't thought to rename it.